ggml-cuda.cu

```diff
@@ -5247,7 +5247,7 @@ static cudaError_t ggml_cuda_cpy_tensor_2d(
     if (src->backend == GGML_BACKEND_CPU) {
         kind = cudaMemcpyHostToDevice;
         src_ptr = (char *) src->data;
-    } else if (src->backend == GGML_BACKEND_GPU) {
+    } else if (src->backend == GGML_BACKEND_GPU || src->backend == GGML_BACKEND_GPU_SPLIT) {
```

Comment:
Does this really work with split tensors? Wouldn't you need to copy each part of the tensor from a different GPU?

Reply:
It works with the current code but in general that function can return unexpected results because the slice would be relative rather than absolute. I added an assert that enforces analogue behavior to `GGML_BACKEND_GPU`.